crates/hyperswitch_connectors/src/connectors/bambora.rs

```diff
@@ -317,18 +311,18 @@ impl
 
     fn build_request(
         &self,
-        req: &types::PaymentsCompleteAuthorizeRouterData,
-        connectors: &settings::Connectors,
-    ) -> CustomResult<Option<services::Request>, errors::ConnectorError> {
-        let request = services::RequestBuilder::new()
-            .method(services::Method::Post)
-            .url(&types::PaymentsCompleteAuthorizeType::get_url(
+        req: &PaymentsCompleteAuthorizeRouterData,
+        connectors: &Connectors,
+    ) -> CustomResult<Option<Request>, errors::ConnectorError> {
+        let request = RequestBuilder::new()
+            .method(Method::Post)
+            .url(&PaymentsCompleteAuthorizeType::get_url(
                 self, req, connectors,
             )?)
-            .headers(types::PaymentsCompleteAuthorizeType::get_headers(
+            .headers(PaymentsCompleteAuthorizeType::get_headers(
```

Comment:
Nit: You can call `attach_default_headers()` as well.

Reply:
Done